exporters/elasticsearch-exporter/src/main/java/io/camunda/zeebe/exporter/ElasticsearchClient.java

```diff
@@ -201,6 +209,60 @@ private boolean putComponentTemplate(final Template template) {
     }
   }
 
+  public boolean putIndexLifecycleManagementPolicy() {
+    try {
+      final var request = new Request("PUT", "/_ilm/policy/" + configuration.retention.policyName);
+      final var requestEntity =
+          buildPutIndexLifecycleManagementPolicyRequest(configuration.retention.minimumAge);
+      request.setJsonEntity(MAPPER.writeValueAsString(requestEntity));
+      final var response = sendRequest(request, PutIndexLifecycleManagementPolicyResponse.class);
+      return response.acknowledged();
+    } catch (final IOException e) {
+      throw new ElasticsearchExporterException(
+          "Failed to put index lifecycle management policy", e);
+    }
+  }
+
+  static PutIndexLifecycleManagementPolicyRequest buildPutIndexLifecycleManagementPolicyRequest(
+      final Duration minimumAge) {
+    return new PutIndexLifecycleManagementPolicyRequest(
+        new Policy(
+            new Phases(new Delete(formatDuration(minimumAge), new Actions(new ObjectMapper())))));
+  }
+
+  /**
+   * Format defined by
+   * https://www.elastic.co/guide/en/elasticsearch/reference/master/api-conventions.html#time-units
+   */
+  static String formatDuration(final Duration duration) {
```

Comment:
Thanks remco. This morning I noticed some errors when trying to use env vars to define the minimum age. The errors happened in mapping it somewhere. So I agree let's simplify by using strings. ES can handle the details

Reply:
I've applied the changes, and tested changing the config option using an ENV VAR:
<img width="774" alt="Screenshot 2023-03-28 at 09 22 15" src="https://user-images.githubusercontent.com/3511026/228159550-98a15162-638d-43db-96df-3d2fce88152f.png">

Instead of 30 days, the minimum age is now 2 minutes.
<img width="1415" alt="Screenshot 2023-03-28 at 09 22 06" src="https://user-images.githubusercontent.com/3511026/228159541-df351640-5d8a-4cf2-b3c2-7b44a04e48cf.png">